docs/latest.json

```diff
@@ -1 +1 @@
bits":{"type":"string","description":"SSZ hexadecimal","format":"byte"},"previous_justified_checkpoint":{"$ref":"#/components/schemas/Checkpoint"},"current_justified_checkpoint":{"$ref":"#/components/schemas/Checkpoint"},"finalized_checkpoint":{"$ref":"#/components/schemas/Checkpoint"}}},"Fork":{"type":"object","properties":{"previous_version":{"type":"string","description":"Bytes4 hexadecimal","format":"byte"},"current_version":{"type":"string","description":"Bytes4 hexadecimal","format":"byte"},"epoch":{"type":"string","format":"uint64"}}},"PendingAttestation":{"type":"object","properties":{"aggregation_bits":{"type":"string","description":"SSZ hexadecimal","format":"byte"},"data":{"$ref":"#/components/schemas/AttestationData"},"inclusion_delay":{"type":"string","format":"uint64"},"proposer_index":{"type":"string","format":"uint64"}}},"Validator":{"type":"object","properties":{"pubkey":{"type":"string","description":"Bytes48 hexadecimal","format":"byte"},"withdrawal_credentials":{"type":"string","description":"Bytes32 hexadecimal","format":"byte"},"effective_balance":{"type":"string","format":"uint64"},"slashed":{"type":"boolean"},"activation_eligibility_epoch":{"type":"string","format":"uint64"},"activation_epoch":{"type":"string","format":"uint64"},"exit_epoch":{"type":"string","format":"uint64"},"withdrawable_epoch":{"type":"string","format":"uint64"}}},"ValidatorsRequest":{"required":["pubkeys"],"type":"object","properties":{"epoch":{"type":"string","format":"uint64"},"pubkeys":{"type":"array","items":{"type":"string","description":"Bytes48 hexadecimal","format":"byte"}}}},"ValidatorWithIndex":{"type":"object","properties":{"pubkey":{"type":"string","description":"Bytes48 hexadecimal","format":"byte"},"validator_index":{"type":"integer","format":"int32"},"balance":{"type":"string","format":"uint64"},"validator":{"$ref":"#/components/schemas/Validator"}}},"SyncStatus":{"type":"object","properties":{"startingSlot":{"type":"string","format":"uint64"},"currentSlot":{"type":"string","format":"uint64"},"highestSlot":{"type":"string","format":"uint64"}}},"SyncingResponse":{"type":"object","properties":{"is_syncing":{"type":"boolean"},"sync_status":{"$ref":"#/components/schemas/SyncStatus"}}},"BeaconValidators":{"type":"object","properties":{"validators":{"type":"array","items":{"$ref":"#/components/schemas/ValidatorWithIndex"}},"total_size":{"type":"integer","format":"int64"},"next_page_token":{"type":"integer","format":"int32"}}},"ValidatorDuties":{"type":"object","properties":{"validator_pubkey":{"type":"string","description":"Bytes48 hexadecimal","format":"byte"},"aggregator_modulo":{"type":"integer","format":"int32"},"validator_index":{"type":"integer","format":"int32"},"attestation_committee_index":{"type":"integer","format":"int32"},"attestation_committee_position":{"type":"integer","format":"int32"},"block_proposal_slots":{"type":"array","items":{"type":"string","format":"uint64"}},"attestation_slot":{"type":"string","format":"uint64"}}}}}}
```

Comment:
If this file is generated, I don't think it should be committed. Instead, add the necessary steps to generate this file and include those as an earlier job in the build, similar to how you're generating the `openapi.json` file and persisting it via workspaces.

Reply:
this file is part of the docs folder, so here it was updated by running the branch CI (see commit) and left here on purpose to show it works.